gma/src/ios/native_ad_internal_ios.mm

```diff
@@ -119,6 +120,18 @@
   firebase::MutexLock lock(mutex_);
   native_ad_ = ad;
 
+  NativeAdImageInternal icon_internal;
+  icon_internal.native_ad_image = ad.icon;
+  GmaInternal::InsertNativeInternalImage(this, icon_internal, std::string("icon"), true );
```

Comment:
Similar to comment above, why can't we just pass "icon" directly?

Reply:
Ack.